test/com/ableton/VirtualEnvTest.groovy

```diff
@@ -19,5 +21,30 @@ class VirtualEnvTest extends BasePipelineTest {
 
     this.script = loadScript('test/resources/EmptyPipeline.groovy')
     assertNotNull(script)
+    script.env = ['BUILD_NUMBER': 1]
+
+    helper.registerAllowedMethod('isUnix', [], JenkinsMocks.isUnix)
+    helper.registerAllowedMethod('pwd', [Map.class], JenkinsMocks.pwd)
+    helper.registerAllowedMethod('sh', [String.class], JenkinsMocks.sh)
+  }
+
+
+  @Test
+  void newObject() throws Exception {
+    def venv = new VirtualEnv(script, 'python2.7')
+    assertNotNull(venv)
+    assertNotNull(venv.script)
+    assertNotNull(venv.destDir)
+  }
```

Comment:
That test will only pass if I do have `python2.7` on my machine, right?

Reply:
Actually the tests will fail everywhere because I didn't update `VirtualEnvTest` yet. 😅 I wanted to push my new implementation of `sh` with the mocked output the script return values, but I'll fix this now.

Reply:
Is this something you still want to do in this PR?